extensions/amp-sticky-ad/0.1/amp-sticky-ad.js

```diff
@@ -27,14 +27,14 @@ class AmpStickyAd extends AMP.BaseElement {
   constructor(element) {
     super(element);
 
+    /** @const @private {!../../../src/service/vsync-impl.Vsync} */
+    this.vsync_ = this.getVsync();
```

Comment:
`vsync` is fine - it's by definition a window-wide service since we always want to to align reads and writes.